src/xcode/ENA/ENAUITests/TicketValidation/ENAUITests_19_TicketValidation.swift

```diff
@@ -0,0 +1,36 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import XCTest
+
+class ENAUITests_19_TicketValidation: CWATestCase {
+	var app: XCUIApplication!
+	
+	override func setUpWithError() throws {
+		try super.setUpWithError()
+		continueAfterFailure = false
+		app = XCUIApplication()
+		setupSnapshot(app)
```

Comment:
```suggestion
```
No snapshots/screenshots in the tests so no setup needed.

Reply:
aye aye